synapse/http/matrixfederationclient.py

```diff
@@ -406,7 +404,14 @@ def __init__(
         self.clock = hs.get_clock()
         self._store = hs.get_datastores().main
         self.version_string_bytes = hs.version_string.encode("ascii")
-        self.default_timeout = 60
+        self.default_timeout = hs.config.federation.client_timeout_ms / 1000
+
+        self.max_long_retry_delay = hs.config.federation.max_long_retry_delay_ms / 1000
```

Comment:
In any case, let's label the units

```suggestion
        self.max_long_retry_delay_seconds = hs.config.federation.max_long_retry_delay_ms / 1000
```